### DISCUSSION TITLE
About cannot plot the Residuals_predicted figure

### DISCUSSION THREAD

**Discussion by KYRA-ma** [lululxvi/deepxde#876] [General]
Dear @lululxvi ,

Recently I have been watching the course which u have given lessons in Tianyuan Mathematical Centre, and I just installed the deepxde, and run the project fnn_pde in the https://github.com/lululxvi/tutorials/tree/master/20211208_deep_learning_handson
but i only got two figures and the third figure u have mentioned in the video is missing. I don't know how to fix it. Could you provide some help for me, since I'm completely new to it.
Hope for your apply.

**Comment by lululxvi** [comment]
See https://deepxde.readthedocs.io/en/latest/demos/pinn_forward/poisson.1d.dirichlet.html

**Reply by KYRA-ma** [reply]
thx! I made it!
